flight/UAVObjects/uavobjectmanager.c

```diff
@@ -54,10 +55,18 @@ typedef void* InstanceHandle;
 struct ObjectEventEntry {
 	struct pios_queue         *queue;
 	UAVObjEventCallback       cb;
-	uint8_t                   eventMask;
+	uint8_t                   hasThrottle : 1;
+	uint8_t                   eventMask : 7;
 	struct ObjectEventEntry * next;
 };
 
+struct ObjectEventEntryThrottled {
+	struct ObjectEventEntry   entry;
```

Comment:
Should add a comment that this member MUST be the first or everything will break.